docs/resources.md

```diff
@@ -648,11 +648,18 @@ The Cluster resource has the following parameters:
     certificate.
 -   `cadata` (required): holds PEM-encoded bytes (typically read from a root
     certificates bundle).
+-   `clientKeyData`: contains PEM-encoded data from a client key file 
+        for TLS 
+-   `clientCertificateData`: contains PEM-encoded data from a client cert file for TLS
+
 
 Note: Since only one authentication technique is allowed per user, either a
 `token` or a `password` should be provided, if both are provided, the `password`
 will be ignored.
 
+`ClientKeyData` and `ClientCertificateData` are only required if `token` or 
```

Comment:
I think these should be `clientKeyData` and `clientCertificateData`?

Reply:
I already did that. https://github.com/tektoncd/pipeline/blob/bd5b2fbc5cc915024cd5ba86de5ffb1a729951b9/docs/variables.md

Reply:
Yeah, will correct this typo. Thanks.